packages/braid-design-system/src/lib/components/Inline/Inline.screenshots.tsx

```diff
@@ -342,5 +342,40 @@ export const screenshots: ComponentScreenshot = {
         </Inline>
       ),
     },
+    {
+      label:
+        'Test - truncation should work an (all examples should truncate below)',
```

Comment:
```suggestion
        'Test - truncation should work (all examples should truncate below)',
```
think this is the expected label?